python/feathub/feature_tables/tests/test_file_system_source_sink.py

```diff
@@ -42,10 +48,28 @@ def test_local_file_system_json_source_sink(self):
         sink_path = tempfile.NamedTemporaryFile(dir=self.temp_dir, suffix=".json").name
         self._test_file_system_source_sink(sink_path, "json")
 
-    def _test_file_system_source_sink(self, path: str, data_format: str):
+    def test_local_file_system_protobuf_source_sink(self):
+        sink_path = tempfile.NamedTemporaryFile(
+            dir=self.temp_dir, suffix=".protobuf"
+        ).name
+        self._test_file_system_source_sink(
+            sink_path,
+            "protobuf",
+            {
+                PROTOBUF_JAR_PATH_CONFIG: get_protobuf_jar_path(),
+                PROTOBUF_CLASS_NAME_CONFIG: "org.feathub.proto.FileSystemTestMessage",
```

Comment:
`message.proto` is currently placed in "java/feathub-test-proto/src/proto" and its compilation depends on the code specified in ./java/feathub-test-proto/pom.xml.

It will be great if we can provide a more pythonic end-to-end solution that allows user to specify a protobuf file in python space with built-in infra to compile and use the protobuf in FeatHub. Is it feasible? I am wondering how pyflink or pyspark allows user to use protobuf.

BTW, it might be useful to checkout how `python/setup.py` compiles `python/feathub/common/protobuf/value.proto` with `generate_py_protobuf`, and re-use as much infra as possible.